about.html

```diff
@@ -0,0 +1,201 @@
+        </nav>
+      </div>
+
+    </div>
+
+  </header>
+
+  <main>
+    <div class="container">
+      <div class="hero-section flex-column" id="hero-section">
+        <div class="shapes">
+          <img src="shapes/shape7.png" alt="hexagon" class="shape7">
+          <img src="shapes/why-shape-dot.png" alt="shapes dots" class="shape-dot">
+          <img src="shapes/hexagon.png" alt="" class="hexagon">
+        </div>
+        <div class="hero-section-about">
+          <h3 class="hero-first-title-about flex-column">
+            "Become a Pro Web Developer"
+          </h3>
+          <h1 class="hero-title flex-column hero-title-about">
+            Learn Web Development
+          </h1>
+          <div class="hero-desc-about flex-column">
+            <p>
+              Get access to 2700+ web development courses taught by top instructors
+              on our online learning platform. Our courses cover topics for
+              beginners to advanced level.
+            </p>
+            <p>
+              Our top instructors come from leading tech companies
+              and offer expertise in HTML, CSS, JavaScript, and full-stack development.
+              Get started on your web development journey today with our courses.
+            </p>
+          </div>
+        </div>
+        <div class="contactus">
+          <p>
+            Get in touch with us to learn more about our online web development courses.
+            Email us at <a href="#">bahatizirimwabagabo@gmail.com</a> or visit us in Gisenyi city, Rwanda.
+          </p>
+        </div>
+
+      </div>
+    </div>
+    <div class="our-project">
+      <div class="container">
+        <div class="ourproject-desc">
+          <div class="title flex-column">
+            <h4> Explore the Latest Trends and Best Practices in 2023</h4>
+
+            <p>Discover the latest trends and best practices in web development
+              with our Creative Web Development program. Learn from industry experts
+              and stay ahead of the game with the latest web technologies
+            </p>
+          </div>
+          <div class="our-project-img">
+            <img src="images/best-project.jpg" alt="web development">
+          </div>
+
+        </div>
+
+
+      </div>
+
+    </div>
+
+    <div class="latest-projects">
+      <div class="container">
+        <div class="title flex-column">
+          <h4> Explore the Latest Trends Project</h4>
+          <hr>
+          <p>
+            Stay up-to-date with the latest trends in web development with our exploratory project.
+            Learn new technologies and techniques from expert instructors
+          </p>
+        </div>
+
+      </div>
+
+      <div class="latest-project-img flex">
+        <div class="images-one">
+          <img src="images/project-2.jpg" alt="project-2">
+        </div>
+        <div class="images-one">
+          <img src="images/project-3.jpg" alt="project-3">
+        </div>
+      </div>
```

Comment:
Nice work 💯 but make sure that these images are the same sizes to keep the design consistent, take a look at the differences 👇🏽 

## Your design

![Screenshot 2023-04-28 at 03 35 35](https://user-images.githubusercontent.com/45739963/235084536-58d2949c-f9c5-44e0-a070-c52bbb88265c.png)

## Figma

![Screenshot 2023-04-28 at 03 36 13](https://user-images.githubusercontent.com/45739963/235084636-5e3c5cb1-c85d-430f-aea0-e91ed26f4070.png)

Once again and valuable for the whole project 📁 make sure to increase the font-size and adjust the spacing. 🤓

Reply:
i fix the font-size to every component that require the to increase the font-size,